tests/pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+addopts = --strict-markers -m 'not nonfree'
+markers =
+    nonfree: marks tests as requiring nonfree (i.e., having restrictive licenses) dependencies
```

Comment:
I agree, and I changed to this instead. A summary of the reasons is in [the commit message](https://github.com/tulip-control/polytope/pull/98/commits/2af689350226402de6305644dbd7c97f6050c4f3):

> skipif() has the advantage of explicitly warning the user if tests
> are not included due to gurobipy not being installed. This is entirely
> similar to behavior in plot_test.py with matplotlib.
>
> This changeset replaces the `nonfree` marker, which was not friendly
> to newcomers: they would need to know to `pytest -m nonfree` if they
> wanted to test with Gurobi. Otherwise, `pytest` (without the marker)
> would silently not include those tests.
>
> The advantages of using the `nonfree` marker are:
> 1. emphasizing to the user that tests involve behavior by code without
>   open source,
> 2. failing if the corresponding packages (gurobipy, possibly others in
>   the future) are not installed.
> However, users who know enough to use `pytest -m nonfree` likely are
> aware of Gurobi's license and likely would notice a warning when tests
> are running, so indeed, skipif() is the best choice.